storage/innobase/srv/srv0srv.cc

```diff
@@ -1479,6 +1485,29 @@ srv_export_innodb_status(void)
 
 	export_vars.innodb_buffer_pool_pages_dirty = flush_list_len;
 
+	export_vars.innodb_buffer_pool_pages_made_young = stat.n_pages_made_young;
+	export_vars.innodb_buffer_pool_pages_made_not_young = stat.n_pages_not_made_young;
```

Comment:
We try to keep the InnoDB source code lines at most 79 characters long when TAB is counted as 8 characters.